docs/docs/how-tos/upgrade-kubernetes-version.md

```diff
@@ -0,0 +1,155 @@
+---
+id: kubernetes-version-upgrade
+title: How to upgrade the cluster's Kubernetes version
+description: A basic overview on how to upgrade your cluster Kubernetes version
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Overview
+
+Nebari runs on Kubernetes under the hood, and as administrators of this Kubernets cluster, one of the maintenance tasks required from time to time includes upgrading the version of your Kubernetes cluster. Each of the different cloud providers that Nebari can run on, has their own release cycle and support windows for their flavor of Kubernetes. That said, they tend to follow the [official Kubernetes release cycle](https://kubernetes.io/releases/).
+
+The Nebari development team tries to stay ahead of this by supporting the latest version when possible. However given that many Kubernetes releases comes with a set of deprecations that potentially affect Nebari and downstream plugins, there is an enforced [`HIGHEST_SUPPORTED_K8S_VERSION`](https://github.com/nebari-dev/nebari/blob/91792952b67074b5c15c3b4009bde5926ca4ec6b/src/_nebari/constants.py#L11) allowed.
+
+
+:::note
+This `HIGHEST_SUPPORTED_K8S_VERSION` is, at times, a minor version or two behind the officially released Kubernetes version.
+:::
+
+Many cloud providers enable users to automatically upgrade their Kubernetes cluster (control plane), however given the potential for deprecations and other changes, Nebari deployed to these cloud have this feature automatically disabled. 
+
+This upgrade process bumps the version of the control plane along with all node groups. 
+
+:::warning
+Upgrading the kubernetes version of the node groups will cause downtime so please plan accordingly. We also recommend [backing up your data](./manual-backup.md) before starting this upgrade process.
+:::
+
+:::warning
+Kubernetes versions can only be upgraded one minor version at a time. If you're running on 1.24, and need to upgrade to 1.26, you will first need to upgrade to 1.25.
+:::
+
+:::warning
+Downgrading to a lower version of Kubernetes is dangerous and we strongly advise against it!
+:::
+
+<Tabs>
+  
+<TabItem label="GCP" value="gcp" default="true">
+
+Google Kubernetes Engine (GKE) cut their own platform specific version of Kubernetes that usually look something like: `1.26.7-gke.500`; this corresponds to a Kubernetes version of `1.26.17`.
```

Comment:
Is this a typo or does 1.26.7-gke.500 really correspond to 1.26.17?

Reply:
Typo :)